integrations/extensions/starter-kits/language-model-conversational-search/README.md

```diff
@@ -1,8 +1,12 @@
 # Language Model Conversational Search starter kit
 
-You can use this starter kit to configure language models with Watson Assistant for conversational search. This starter kit shows two ways to do this. The first is an example of [how the IBM watsonx tech preview language model and Watson Discovery search can be used to provide a personalized response to a customer](#example-1-connect-your-assistant-to-a-watsonx-language-model-via-custom-extensions). A second example shows how to use [semantic search output as input to OpenAI](#example-2-connect-your-assistant-to-hugging-face-milvus-and-openai-via-custom-extensions).
+This starter kit has multiple examples of how to configure language models with Watson Assistant for conversational search.
 
-The [prerequisites](#prerequisites) are the same for both examples.
```

Comment:
"both" should be "all" now

Reply:
Agree. This is the old text, it has already been replaced.